Examples/RPC/go/main.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/golang/protobuf/proto"
+	"google.golang.org/genproto/googleapis/cloud/language/v1"
+	"google.golang.org/genproto/googleapis/rpc/status"
+)
+
+func main() {
+	request := &language.AnalyzeEntitiesRequest{
+		Document: &language.Document{
+			Type: language.Document_PLAIN_TEXT,
+			Source: &language.Document_Content{
+				Content: "The rain in Spain stays mainly in the plain.",
+			},
+		},
+	}
+	data, err := proto.Marshal(request)
+	if err != nil {
+		panic(err)
+	}
+
+	url := "https://language.googleapis.com/$rpc/" +
+		"google.cloud.language.v1.LanguageService/AnalyzeEntities"
+	req, err := http.NewRequest("POST", url, bytes.NewBuffer(data))
+	if err != nil {
+		panic(err)
+	}
+	req.Header.Set("X-Goog-Api-Key", os.Getenv("GOOGLE_API_KEY"))
```

Comment:
Please use a library. Otherwise the code is too messy.

Reply:
Thanks for your thoughts on this. I think our intent for these examples should be to make them as low-level as possible, particularly in terms of relying only on standard libraries. These would be the examples that people use to write their own client libraries. We can/will add higher-level examples elsewhere, maybe in language-specific repos.